api/app/tests/apps/signals/test_task_anonymize_reporters.py

```diff
@@ -42,12 +45,13 @@ def test_anonymize_reporters(self):
         anonymize_reporters(days=1)
 
         self.assertEqual(Reporter.objects.count(), 4)
-        self.assertEqual(Reporter.objects.filter(is_anonymized=True).count(), 4)
+        self.assertEqual(Reporter.objects.filter(Q(email_anonymized=True) | Q(phone_anonymized=True)).count(), 4)
```

Comment:
Should this be an `&`? we want both email and phone removed from the data.

Also below, I will not repeat this comment.

Reply:
It could be that there was no `email` or `phone` so it can be that both or either one are anonymized. That's why I used an OR.